homeassistant/components/stream/fmp4utils.py

```diff
@@ -9,6 +9,8 @@
 if TYPE_CHECKING:
     from io import BufferedIOBase
 
+    from .core import Orientation
```

Comment:
I don't think this can simply be inside TYPE_CHECKING

Reply:
Whoops